tests/test_sensor.py

```diff
@@ -262,3 +262,37 @@ def test_remove_callbacks(self, ss, sensors, add_callback, remove_callback):
         ss.remove(sensors[0])
         remove_callback.assert_not_called()
         add_callback.assert_not_called()
+
+
+def test_aggregate_sensor(mocker, ss, sensors):
+    """Test operation of AggregateSensor.
+
+    Create a simple derived class, mock out the _update_aggregate function, and
+    test that (1) the target is properly assigned, and (2) the update function
+    is called at all appropriate occasions.
+    """
+
+    class MyAgg(AggregateSensor):
+        def _update_aggregate(self, *args):
+            pass
+
+    my_agg = MyAgg(target=ss, sensor_type=int, name="good-bad-ugly")
+    assert my_agg.target == ss
```

Comment:
```suggestion
    assert my_agg.target is ss
```
It should reference the original and not be a copy (although I think `==` will be by object identity so practically it might not matter).